cloudmailer/cloudmailer.py

```diff
@@ -202,21 +199,11 @@ def getProjectID(self, tenant_name):
 
     def getProjectInfo(self, tenant_id):
         # Get project info. Name, memberemails
-        project = {}
-        emails = []
-        users = []
-
         if tenant_id is None:
-            print ("Undefined input project ID while retrieving project data! Possibly trying to retrieve project data from the wrong domain.")
-            return {"name": None, "emails": emails, "servers": []}
-        users = self.getRoleAssignment(tenant_id)
-        for user in users:
-            email = self.getUserEmail(user)
-            if email:
-                emails.append(email)
-
-        if len(emails) == 0:
-            print(tenant_id + " does not have any emails")
+            print ("Undefined input project ID while retrieving project data! " \
+                   "Possibly trying to retrieve project data from the wrong domain.")
+            return {"name": None, "emails": [], "servers": []}
```

Comment:
Should this ever happen? Maybe this if statement could be removed?

Reply:
The only case that came to my mind is handling some extremely old / erroneous virtual machine, for which the project id is set to None due to some corner case. I guess it is always good to check anyway